drivers/dac/dac_mcp4725.c

```diff
@@ -139,7 +139,7 @@ static const struct dac_driver_api mcp4725_driver_api = {
 	};								\
 									\
 	DEVICE_DT_INST_DEFINE(index, dac_mcp4725_init,			\
-			    device_pm_control_nop,			\
+			    NULL,					\
```

Comment:
Can you split this out into its own PR.  This is a bug that should get fixed in 2.6.0

Reply:
I've removed this change from this PR

Reply:
Please do submit a new PR with this in it.

Reply:
PR created #35265